# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/clients/manager/ClientSource.cpp
# Commit: abfad4bc648f
# Full Hash: abfad4bc648f9669b84dda2dc87a729d49dcb45f
# Author: Tim Huang <tihuang@mozilla.com>
# Date: 2020-05-27 16:14:31
# Regressor Bug: 1641153
# File Overlap Count: 1
# Description:
#   Bug 1641153 - Change the MOZ_DIAGNOSTIC_ASSERT to MOZ_ASSERT in ClientSource.cpp r=dimi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D77001
# ==============================================================================

diff -r 636b70578d73 -r abfad4bc648f dom/clients/manager/ClientSource.cpp
--- a/dom/clients/manager/ClientSource.cpp	Wed May 27 09:17:47 2020 +0000
+++ b/dom/clients/manager/ClientSource.cpp	Wed May 27 08:53:45 2020 +0000
@@ -262,10 +262,9 @@
   // continue to inherit the SW as well.  We need to avoid triggering the
   // assertion in this corner case.
   if (mController.isSome()) {
-    MOZ_DIAGNOSTIC_ASSERT(spec.LowerCaseEqualsLiteral("about:blank") ||
-                          StringBeginsWith(spec, NS_LITERAL_CSTRING("blob:")) ||
-                          StorageAllowedForWindow(aInnerWindow) ==
-                              StorageAccess::eAllow);
+    MOZ_ASSERT(spec.LowerCaseEqualsLiteral("about:blank") ||
+               StringBeginsWith(spec, NS_LITERAL_CSTRING("blob:")) ||
+               StorageAllowedForWindow(aInnerWindow) == StorageAccess::eAllow);
   }
 
   nsPIDOMWindowOuter* outer = aInnerWindow->GetOuterWindow();
